packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -139,6 +139,21 @@ const EVENTS = {
     event: 'reset_password_confirmed_continue',
   },
 
+  // Reset password with recovery key verified
+  'screen.settings.reset-password-with-recovery-key-verified': {
```

Comment:
I know in your other PR we referenced `settings` here but we should just use the existing events if we can, like `flow.reset-password.submit` for example, with that `entrypoint_variation: 'react',` option as the delta, for easier 1:1 event comparison and less friction when we remove content-server (else we have to change them all at that point - I know different teams look at our metrics). If leaving them in the "settings" group is preferred we should run it by Wil.

Reply:
@LZoog Is `flow.reset-password.submit` used for all form submissions in the password reset flow (not just the initial submit for `ResetPassword` page)?

Reply:
@vpomerleau It shouldn't be because then the metrics would have less value if we couldn't differentiate which click actually fired the event, but I would have to go into each reset password JS file to check. 🙈 Our metrics events are kind of hard to tease out because I think the word "flow" is applied automatically in places as well as the view event and we've got some regex in that `amplitude.js` file that are hard to look at as well.